### DISCUSSION TITLE
Auto-find geography (and other information) from coordinates

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5824] [Feature Requests]
**Requested By:** Bern

This involves determining a geography based on a set of coordinates

**Comment by maxpatiiuk** [comment]
Related to https://github.com/specify/specify7/issues/1940

**Comment by emenslin** [comment]
Closing in favor of #3720

**Comment by grantfitzsimmons** [comment]
@emenslin These are two different issues
